sdcm/utils/net.py

```diff
@@ -31,8 +31,8 @@ def get_my_public_ip() -> str:
             result.raise_for_status()
 
             ip_address = result.text.strip()
-            ipaddress.ip_address(ip_address)  # validating that we got IP address
-        except (ValueError, requests.exceptions.RequestException):
+            ipaddress.IPv4Address(ip_address)  # validating that we got IPv4 address
```

Comment:
Cannot we return from this place?

Reply:
changed the code to return as soon it find a valid ip